config.go

```diff
@@ -43,7 +43,7 @@ const (
 	defaultBlockMaxSize      = 375000
 	blockMaxSizeMin          = 1000
 	blockMaxSizeMax          = wire.MaxBlockPayload - 1000
-	defaultBlockPrioritySize = 50000
+	defaultBlockPrioritySize = 0
```

Comment:
If miners prefer not to have a priority area in the block, they can use the `--blockprioritysize` option which exists *precisely* for this purpose.  Having a high priority area is important for the health of the network as it grows.  The default policy should not be changed.